ui.tests/test-module/libs/fixtures/panelcontainer/beforeResetBank.json

```diff
@@ -44,6 +44,6 @@
         "instituteName": "DU"
       }, {}],
       "gender": "Female"
-    }, {}]
+    }]
```

Comment:
@rismehta How did your update version of core PR (with reset data fix) passed, without fixing reset data in reset test case?